dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/blocking/jdbc/DssDatabaseServiceActor.java

```diff
@@ -0,0 +1,75 @@
+package io.github.ztkmkoo.dss.core.actor.blocking.jdbc;
+
+import com.healthmarketscience.sqlbuilder.BinaryCondition;
+import com.healthmarketscience.sqlbuilder.InsertQuery;import com.healthmarketscience.sqlbuilder.InsertSelectQuery;
+import com.healthmarketscience.sqlbuilder.SelectQuery;
+
+import akka.actor.typed.Behavior;
+import akka.actor.typed.javadsl.ActorContext;
+import akka.actor.typed.javadsl.Behaviors;
+import io.github.ztkmkoo.dss.core.message.blocking.jdbc.DssJdbcCommand;
+import lombok.extern.log4j.Log4j;
+
+
+@Log4j
```

Comment:
@JuHyun419 
Hi there,

you can log with context.getLog() in actor
and already we on dependency with logback ^^

Reply:
Oh I'll check!

Reply:
Oh I'll check :)